src/containers/Charts/Chart.js

```diff
@@ -160,18 +158,17 @@ const Chart = ({ solarData, history }) => {
         </div>
       </div>
       <div className="savings-display-bar">
-        {history ?
-          <p>Based on your annual energy usage, this solar system would offset {savingsValue}% per year! </p> :
-          < p > For a true savings estimate, please enter your <Link to="/historical"> home energy data</Link> </p>
+      {solarData.percentOffset ?
+        <p> Based on your annual energy usage, this solar system would offset {solarData.percentOffset} % per year! </p> :
+        <p> For a true savings estimate, please enter your <Link to="/historical"> home energy data</Link> </p>
         }
       </div>
     </div>
   )
 }
 
 const mapStateToProps = (state) => ({
-  solarData: state.solarData.outputs,
-  history: state.solarData.historicalData
+  solarData: state.solarData
 })
 
```

Comment:
these values were changed as the BE API was updated to now return a savings value and the return was formatted as a more clean return object